src/kotlin/example/spring/build.gradle.kts

```diff
@@ -21,12 +21,14 @@ repositories {
 dependencies {
     implementation("org.jetbrains.kotlin:kotlin-reflect")
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core")
+    implementation("org.jetbrains.kotlinx:kotlinx-coroutines-reactor:1.7.3") // Need this guy too, because why would coroutines just need one dependency?
```

Comment:
:)

Reply:
forgot i copied this whole line but it looks great i think
also thanks for spotting it